tests/hopp/test_dispatch.py

```diff
@@ -1023,4 +1044,52 @@ def test_hybrid_dispatch_baseload_heuristic_and_analysis(site):
 
     assert hybrid_plant.grid.time_load_met == pytest.approx(91.9, 1e-2)
     assert hybrid_plant.grid.capacity_factor_load == pytest.approx(94.45, 1e-2)
-    assert hybrid_plant.grid.total_number_hours == pytest.approx(3732, 1e-2)
\ No newline at end of file
+    assert hybrid_plant.grid.total_number_hours == pytest.approx(3732, 1e-2)
+
+def test_dispatch_load_following_heuristic_with_wave(site, subtests):
+    dispatch_options = {'battery_dispatch': 'load_following_heuristic', 'grid_charging': False}
+    wave_battery = {key: technologies[key] for key in ['wave', 'battery', 'grid']}
+
+    for tech in wave_battery.keys():
+        wave_battery[tech]["fin_model"] = default_fin_config
+
+    wave_resource_file = ROOT_DIR / "simulation" / "resource_files" / "wave" / "Wave_resource_timeseries.csv"
+
+    desired_schedule=desired_schedule = 8760*[20]
```

Comment:
Looks like there is an extra `desired_schedule=` here.

Reply:
Thanks, fixed